src/components/AssetsTable/index.jsx

```diff
@@ -564,11 +583,7 @@ export default class AssetsTable extends React.Component {
       />
     );
 
-    return (
-      <React.Fragment>
-        {statusAlert}
-      </React.Fragment>
-    );
+    return statusAlert;
```

Comment:
I don't think we need the React.Fragment here, since it's a single element, but please correct me if I'm wrong.